File: GTTS/INL.TrainingService.Models/GetTrainingEventVettingPreviewBatches_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventVettingPreviewBatches_Result : IGetTrainingEventVettingPreviewBatches_Result
    {
        public int MaxBatchSize { get; set; }
        public int LeahyBatchLeadTime { get; set; }
        public int CourtesyBatchLeadTime { get; set; }
        public List<GetTrainingEventBatch_Item> LeahyBatches { get; set; }
        public List<GetTrainingEventBatch_Item> CourtesyBatches { get; set; }
        public List<GetTrainingEventBatch_Item> LeahyReVettingBatches { get; set; }
        public List<GetTrainingEventBatch_Item> CourtesyReVettingBatches { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-vetting-preview-batches_result.ts


import { GetTrainingEventBatch_Item } from './get-training-event-batch_item';
import { IGetTrainingEventVettingPreviewBatches_Result } from './iget-training-event-vetting-preview-batches_result';

export class GetTrainingEventVettingPreviewBatches_Result implements IGetTrainingEventVettingPreviewBatches_Result {
  
	public MaxBatchSize: number = 0;
	public LeahyBatchLeadTime: number = 0;
	public CourtesyBatchLeadTime: number = 0;
	public LeahyBatches?: GetTrainingEventBatch_Item[];
	public CourtesyBatches?: GetTrainingEventBatch_Item[];
	public LeahyReVettingBatches?: GetTrainingEventBatch_Item[];
	public CourtesyReVettingBatches?: GetTrainingEventBatch_Item[];
  
}






